static/app/views/issueList/actions/actionSet.tsx

```diff
@@ -236,81 +211,75 @@ function ActionSet({
           {t('Unarchive')}
         </Button>
       ) : null}
-      {selectedProjectSlug ? (
-        <Projects orgId={organization.slug} slugs={[selectedProjectSlug]}>
```

Comment:
This is where the remounting was happening. Basically moved all the project logic inside `ResolveActions`